packages/webdriverio/tests/commands/element/setValue.test.js

```diff
@@ -44,3 +44,29 @@ describe('setValue', () => {
         expect(request.mock.calls[3][0].body.text).toEqual('12true[1,2]')
     })
 })
+
+describe('should allow to add value to an input element as workaround for /webdriverio/issues/4936', () => {
```

Comment:
Can we merge this into the `describe` block above?

Reply:
Yes, I'll do it